### DISCUSSION TITLE
OSZ to MP4

### DISCUSSION THREAD

**Discussion by itsfprime** [ppy/osu#16859] [Ideas] [answered]
A challenge in creating Osu! videos is recording gameplay manually. There is no way to speed up this process of recording an osu! replay and uploading it. What if there was a way to convert an osz to an mp4 via the map information and cursor information. I don't think i am thinking this through the whole way, but I think it would be a really neat feature.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/12986

**Reply by itsfprime** [reply]
whoops! I didn't see that, sorry!
